packages/components/src/ActionCell/useLogic/useLogic.ts

```diff
@@ -0,0 +1,41 @@
+import { type MouseEventHandler, useCallback } from 'react';
+
+import { type ActionsCellProps } from '../ActionCell';
+import { type NestedAction, type SingleAction } from '../types';
+
+type UseLogicParams<Action> = ActionsCellProps<Action>;
```

Comment:
https://kaluga-astral.github.io/style-guide/docs/rules/ts#%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D0%BF%D0%B0%D1%80%D0%B0%D0%BC%D0%B5%D1%82%D1%80%D0%BE%D0%B2-generic-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D0%B8%D0%BC%D0%B5%D1%82%D1%8C-%D0%BF%D1%80%D0%B5%D1%84%D0%B8%D0%BA%D1%81-t-%D0%B8-%D1%83%D1%82%D0%BE%D1%87%D0%BD%D1%8F%D1%8E%D1%89%D0%B5%D0%B5-%D1%81%D0%BB%D0%BE%D0%B2%D0%BE